src/v2/Apps/ArtistSeries/Components/ArtistSeriesHeader.tsx

```diff
@@ -24,59 +84,82 @@ const ArtistSeriesHeader: React.FC<ArtistSeriesHeaderProps> = props => {
   const {
-    artistSeries: { title, description },
+    artistSeries: { title, description, artists },
   } = props
   return (
-    <Box>
-      <Grid>
-        <Row>
-          <Col sm={6}>
-            <Flex
-              height="100%"
-              flexDirection="column"
-              justifyContent="space-between"
-            >
-              <Sans size="8" element="h1" unstable_trackIn>
-                {title}
-              </Sans>
-              <Sans lineHeight={1.5} size="3t" pr={1}>
-                {description}
-              </Sans>
-            </Flex>
-          </Col>
-          <Col sm={6}>
-            <Box
-              // FIXME: Use Image
-              bg="black10"
-              width={1}
-              height={400}
-            ></Box>
-          </Col>
-        </Row>
-      </Grid>
-    </Box>
+    <>
+      <Flex
+        position="relative"
+        alignItems="center"
+        justifyContent="center"
+        p={2}
+      >
+        <Flex position="absolute" left={3}>
+          {artists.length && <ArtistInfo artist={artists[0]} />}
+        </Flex>
+        <Sans size="3">Series</Sans>
+      </Flex>
+      <Separator />
+      <Box m={3}>
+        <Grid>
+          <Row>
+            <Col sm={6}>
+              <Flex
+                height="100%"
+                flexDirection="column"
+                justifyContent="space-between"
+              >
+                <Sans size="8" element="h1" unstable_trackIn>
+                  {title}
+                </Sans>
+                <Sans lineHeight={1.5} size="3t" pr={1}>
+                  {description}
+                </Sans>
+              </Flex>
+            </Col>
+            <Col sm={6}>
+              <Box
+                // FIXME: Use Image
+                bg="black10"
+                width={1}
+                height={400}
+              ></Box>
+            </Col>
+          </Row>
+        </Grid>
+      </Box>
+    </>
   )
 }
 
 const ArtistSeriesHeaderSmall: React.FC<ArtistSeriesHeaderProps> = props => {
   const {
-    artistSeries: { title, description },
+    artistSeries: { title, description, artists },
   } = props
   return (
     <>
-      <Box
-        // FIXME: Use Image
-        mx="auto"
-        bg="black10"
-        width={180}
-        height={180}
-      ></Box>
-      <Sans size="8" element="h1" my={1} unstable_trackIn>
-        {title}
-      </Sans>
-      <Sans lineHeight={1.5} size="3t">
-        {description}
-      </Sans>
+      <Box textAlign="center" p={1}>
+        <Sans size="3">Series</Sans>
+      </Box>
+      <Separator />
+      <Box m={3}>
+        <Box
+          // FIXME: Use Image
+          mx="auto"
+          bg="black10"
+          width={180}
+          height={180}
+        ></Box>
+        <Sans size="8" element="h1" my={1} unstable_trackIn>
+          {title}
+        </Sans>
+        {artists.length && <ArtistInfo artist={artists[0]} />}
```

Comment:
I know the expectation is that an Artist Series will only have one artist but I wonder if we should handle the edge cases here when there will be multiple, e.g. looping through the array of artists.

Reply:
If it's just a matter of looping here and below, I'm 👍 with this suggestion. (Might want to quickly ping design to confirm that the resulting visual treament is ok)

Reply:
I saw that on your Eigen PR and think that makes a lot of sense with the mobile design. I briefly tried it out on mobile web and desktop and while mobile web would be fine with some quick adjustments the desktop design begins to have issues with smaller viewports. 

It's not clear to me in the current design how we should treat the case where the multiple artists take up more than half the length of the header on Desktop. E.g. we could stack the `ArtistInfo` components, but I don't think that'd look good. @nicoleyeo any thoughts?

![Screen Shot 2020-07-09 at 1 30 25 PM](https://user-images.githubusercontent.com/4432348/87071737-a69b7200-c1e8-11ea-8c6c-459b42f3dbf7.png)

![Screen Shot 2020-07-09 at 1 29 51 PM](https://user-images.githubusercontent.com/4432348/87071739-a7340880-c1e8-11ea-8065-c7a3f3e452bd.png)

Reply:
Ah right, that "Series" subhead... thanks, these mockups are helpful for illustrating the prob. 

If we can't resolve the design question quickly, then merging and iterating sounds good to me too!

Reply:
Per Slack convo we'll leave the header as-is, i.e. we'll only pass in one artist and not try to render two since the design only supports one.